servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/CoreMessageReceiver.java

```diff
@@ -145,68 +143,60 @@ private CoreMessageReceiver(final MessagingFactory factory,
         this.currentPrefetechedMessagesCount = new AtomicInteger();
         this.entityType = entityType;
 
-        this.timedOutUpdateStateRequestsDaemon = new Runnable() {
-            @Override
-            public void run() {
-                try
-                {
+        this.timedOutUpdateStateRequestsDaemon = () -> {
+            try {
+                if (TRACE_LOGGER.isTraceEnabled()) {
```

Comment:
>if (TRACE_LOGGER.isTraceEnabled()) { [](start = 16, length = 36)

Why are you checking isTraceEnabled()? With SLF4J, you don't have to do this as long as you are using string formatting properly. Remove these isLevelEnabled calls.